images/win/scripts/Tests/Java.Tests.ps1

```diff
@@ -1,6 +1,11 @@
 Describe "Java" {
-    [array]$jdkVersions = (Get-ToolsetContent).java.versions | ForEach-Object { @{Version = $_} }
-    $defaultVersion = (Get-ToolsetContent).java.default
+    $toolsetJava = (Get-ToolsetContent).java
+    $defaultVendor = $toolsetJava.default_vendor
+    $javaVendors = $toolsetJava.vendors
+    $defaultVersion = $toolsetJava.default
+
+    [array]$jdkVersions = ($toolsetJava.vendors | Where-Object {$_.name -eq $defaultVendor}).versions | ForEach-Object { @{Version = $_} }
```

Comment:
```suggestion
    [array]$jdkVersions = ($javaVendors | Where-Object {$_.name -eq $defaultVendor}).versions | ForEach-Object { @{Version = $_} }
```

Reply:
@miketimofeev yep, you are ahead of me ;)